docs/platform/howto/_manage-org-vpc-peering-azure.md

```diff
@@ -0,0 +1,426 @@
+   ```
+
+   :::
+
+1. Create an application object in your AD tenant using the Azure CLI:
+
+   ```bash
+   az ad app create                         \
+     --display-name "NAME_OF_YOUR_CHOICE"   \
+     --sign-in-audience AzureADMultipleOrgs \
+     --key-type Password
+   ```
+
+   This creates an entity to your AD that can be used to log into multiple
+   AD tenants ( `--sign-in-audience AzureADMultipleOrgs` ), but only the
+   home tenant (the tenant the app was created in) has the credentials to
+   authenticate the app.
+
+   :::note
+   Save the `appId` field from the output. It will be referred to as `$user_app_id`.
+   :::
+
+1. Create a service principal for your app object to the Azure subscription that the VNet
+   to be peered is located in:
+
+   ```bash
+   az ad sp create --id $user_app_id
+   ```
+
+   This creates a service principal to your subscription that may be given
+   permissions to peer your VNet.
+
+   :::note
+   Save the `id` field from the JSON output. It will be referred to as `$user_sp_id`.
+   :::
+
+1. Set a password for your app object:
+
+   ```bash
+   az ad app credential reset --id $user_app_id
+   ```
+
+   :::note
+   Save the `password` field from the output. It will be referred to as `$user_app_secret`.
+   :::
+
+1. Find properties of your virtual network:
+
+   - Resource ID
+     - In the [Azure portal](https://portal.azure.com/#home): **Virtual networks** >
+       name of your network > **JSON View** > **id**
+     - Using the Azure CLI:
+
+       ```bash
+       az network vnet list
+       ```
+
+     :::tip
+     The `id` field should have format
+     `/subscriptions/$user_subscription_id/ resourceGroups/$user_resource_group/providers/Microsoft.Network/virtualNetworks/$user_vnet_name`. It will be referred to as `$user_vnet_id`.
+
+   - Azure Subscription ID (the VNet page in the
+     [Azure portal](https://portal.azure.com/#home) > **Essentials** > **Subscription ID**)
+     or the part after `/subscriptions/` in the resource ID. It will be referred to as
+     `$user_subscription_id`.
+   - Resource group name (the VNet page in the [Azure portal](https://portal.azure.com/#home) >
+     **Essentials** > **Resource group**) or the `resourceGroup` field in the output. This
+     will be referred to as `$user_resource_group`.
+   - VNet name (title of the VNet page), or the `name` field from
+     the output. It will be referred to as `$user_vnet_name`.
+
+   :::note
+   Save all the properties for later.
+   :::
+
+1. Grant your service principal permissions to peer.
+
+   The service principal needs to be assigned a role that has the permission for the
+   `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write` action
+   on the scope of your VNet. To limit the amount of permissions the application
+   object and the service principal have, you can create a custom role with just
+   that permission. The built-in network contributor role includes that permission.
```

Comment:
How about something like this here: 

The service principal needs to be assigned a role that includes the `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write` permission at the scope of your VNet. To limit the permissions granted to the application object and the service principal, you can create a custom role with only this permission. The built-in Network Contributor role also includes this permission.